shared/middleware/properties_api.rb

```diff
@@ -126,4 +142,13 @@ class PropertiesApi < Sinatra::Base
   put %r{/v3/(shared|user)-properties/([^/]+)/([^/]+)$} do |_endpoint, _channel_id, _name|
     call(env.merge('REQUEST_METHOD'=>'POST'))
   end
+
+  def self.set_max_property_size_for_test(max_property_size)
+    @@max_property_size = max_property_size
+  end
+
+  def self.reset_max_property_size_for_test
+    set_max_property_size_for_test(DEFAULT_MAX_PROPERTY_SIZE)
+  end
```

Comment:
Are we not able to use stubs in test instead of exposing test-specific methods? i.e. here we do
```
def max_property_size
  DEFAULT_MAX_PROPERTY_SIZE
end
```
in tests (not exactly sure of syntax, but something like:
```
PropertiesApi.any_instance.stubs(:default_max_property_size).returns(10)